specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3790,60 +3773,197 @@
+              "$ref": "#/definitions/MicrosoftSecurityIncidentCreationAlertRuleCommonProperties"
             }
           ],
-          "description": "Filter alert rule template properties",
+          "description": "MicrosoftSecurityIncidentCreation rule template properties",
           "required": [
             "displayName",
             "description",
             "createdDateUTC",
             "status",
             "alertRulesCreatedByTemplateCount",
-            "filterProduct"
+            "productFilter"
           ],
           "x-ms-client-flatten": true
         }
       },
       "type": "object",
-      "x-ms-discriminator-value": "Filter"
+      "x-ms-discriminator-value": "MicrosoftSecurityIncidentCreation"
     },
-    "FilterAlertRuleTemplateProperties": {
-      "description": "Filter alert rule template property bag.",
+    "MicrosoftSecurityIncidentCreationAlertRuleCommonProperties": {
+      "description": "MicrosoftSecurityIncidentCreation rule common property bag.",
       "properties": {
-        "filterProduct": {
-          "description": "The filter product name for this template rule.",
-          "type": "string"
+        "productFilter": {
+          "description": "The alerts' productName on which the cases will be generated",
+          "enum":[
+          	"Microsoft Cloud App Security",
+          	"Azure Security Center",
+          	"Azure Advanced Threat Protection",
+          	"Azure Active Directory Identity Protection"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+          	"modelAsString": true,
+          	"name": "MicrosoftSecurityProductName"
+          }
         },
-        "filterSeverities": {
-          "description": "the alert’s severities on which the cases will be generated",
+        "severitiesFilter": {
+          "description": "the alerts' severities on which the cases will be generated",
           "items": {
             "$ref": "#/definitions/AlertSeverity"
           },
           "type": "array"
         },
-        "filterTitles": {
-          "description": "the alert’s titles on which the cases will be generated",
+        "displayNamesFilter": {
+          "description": "the alerts' displayNames on which the cases will be generated",
           "items": {
             "type": "string"
           },
           "type": "array"
         }
       },
+      "required": [
+      	"productFilter"
+      ],
+      "type": "object"
+    },
+    "FusionAlertRule": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/AlertRule"
+        }
+      ],
+      "description": "Represents Fusion alert rule.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FusionAlertRuleProperties",
+          "description": "Fusion alert rule properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "Fusion"
+		},
```

Comment:
Indentation is messed up in a few places in this file (convert tabs to spaces)

Reply:
👍
formatted the json and sorted the parameters & definition sections